lib/rubocop/cop/performance/unfreeze_string.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Performance
+      # In Ruby 2.3 or later, Use unary plus operator to unfreeze a string
+      # literal instead of `String#dup`.
+      # Unary plus operator is faster than `String#dup`
+      #
+      # @example
+      #   # bad
+      #   "string".dup
+      #   String.new
```

Comment:
Thanks!
I'll add `String.new('')` to bad case.
And I'll add checks for `String.new`, `String.new('')` and `String.new('something')`.